src/styles/mainClassDefinitions.less

```diff
@@ -174,6 +174,40 @@
 	padding-bottom: 28px;
 }
 
+#signInErrorDescription {
```

Comment:
You'll want to put these id css definitions in another file since this one's for class definitions. On a side note we should look at the way we're organizing our less files and see if there's a neater way.